extensions/ipynb/src/serializers.ts

```diff
@@ -224,17 +224,33 @@ type JupyterOutput =
 	| nbformat.IError;
 
 function convertStreamOutput(output: NotebookCellOutput): JupyterOutput {
-	const outputs = output.items
+	const outputs: string[] = [];
+	output.items
 		.filter((opit) => opit.mime === CellOutputMimeTypes.stderr || opit.mime === CellOutputMimeTypes.stdout)
 		.map((opit) => convertOutputMimeToJupyterOutput(opit.mime, opit.data as Uint8Array) as string)
-		.reduceRight<string[]>((prev, curr) => prev.concat(curr), []);
+		.forEach(value => {
+			// Ensure each line is a seprate entry in an array (ending with \n).
+			const lines = value.split('\n');
```

Comment:
Made the changes as per the comments here https://github.com/microsoft/vscode/issues/129370#issue-952511326

Reply:
for the split, we do want to handle CRLF?

Reply:
I don't think we need to (this is how we used to do it & based on what I know this is what Jupyter also does, - code was borrowed from Jupyter)

Reply:
@rchiodo /cc